### PR TITLE
proposed fix for safari chechbox bug

### PR DESCRIPTION
**Pull request by doebi**

### CODE REVIEW

File: components/Checkbox.vue [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <template>
-  <label class="checkbox" :class="theme">
-    <input :value="value" @input="$emit('input', !value);" type="checkbox">
-    <div class="checkmark" :class="{'selected': value}">
-      <svg v-if="value" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 230 200"><path d="M20 130l40 40L200 30" stroke-width="25" fill="none"/></svg>
+  <label class="checkbox" :class="theme" :for="id">
+    <input v-model="dvalue" :id="id" type="checkbox">
```
Comment (tobias-zucali) [inline components/Checkbox.vue]: label "for" is not needed if the input is inside the label, we could skip that here

File: components/Checkbox.vue [hunk 2]
```diff
@@ -11,6 +11,22 @@
 <script>
 export default {
   props: ["value", "theme"],
+  data: () => {
+    return {
+      dvalue: false
```
Comment (tobias-zucali) [inline components/Checkbox.vue]: Shouldn't `dvalue` be initialised with `Boolean(value)`? Now the checkbox can never be initially checked.

Comment (doebi) [reply components/Checkbox.vue]: It is. Check line 20.

File: components/Checkbox.vue (hunk 1 above)
Comment (doebi) [reply components/Checkbox.vue]: done.

File: components/Checkbox.vue (hunk 2 above)
Comment (tobias-zucali) [reply components/Checkbox.vue]: Ah… hoped the initial props are available in the data function, like in a constructor…

Comment (doebi) [reply components/Checkbox.vue]: No, they aren't. Only in the life-cycle hooks i have access to the passed-in props. So "created" or "mounted" can more be referred to as a constructor.

**Pull request merged**